string script_select
string type_select
float32 turn_velocity
float32 move_velocity
float32 total_time
float32 square_length
